client/src/components/App/AppToolbar/TopToolbar/useTopToolbar.tsx

```diff
@@ -0,0 +1,59 @@
+import Swal from 'sweetalert2';
+import axios from 'Utils/axios';
+
+import useStyles from './TopToolbarStyles';
+
+export interface useInteractionsTabInput {
+    setIsActive: (isActive: boolean) => void;
+};
+
+
+export interface useTopToolbarOutcome  {
+    getUserActivityStatus: (userId: string) => void;
+    setUserActivityStatus: (userId: string, isActive: boolean) => void;
+}
+
+const useTopToolbar = (props: useInteractionsTabInput) :  useTopToolbarOutcome => {
+
+    const classes = useStyles({});
+
+    const getUserActivityStatus = (userId: string) => {
+        axios.get(`/usersInfo/userActivityStatus?userId=${userId}`)
+        .then((result) => { 
+            props.setIsActive(result.data.isActive)
+        }).catch(() => {
+            Swal.fire({
+                title: 'לא הצלחנו לקבל את הסטטוס הנוכחי שלך',
+                icon: 'error',
+                customClass: {
+                    title: classes.swalTitle
+                },
+            });
+        });
+    }
+
+    const setUserActivityStatus = (userId: string, isActive: boolean) => {
+        axios.post('usersInfo/updateIsUserActive', {
+            id: userId,
```

Comment:
dont send user id implicitly through a request

Reply:
the user id is updated automatically according to the token in the server, you can access it through _response.locals_ as it is done in other cases

Reply:
Fixed